lib/data/dao/recipe_dao.dart

```diff
@@ -1,15 +1,27 @@
+import "package:collection/collection.dart";
 import 'package:drift/drift.dart';
 import 'package:lunch_me/data/database.dart';
 import 'package:lunch_me/data/exceptions.dart';
 import 'package:lunch_me/data/tables.dart';
 import 'package:validators/validators.dart';
 
+import '../../model/recipe_filters.dart';
```

Comment:
Lets import also internal packages with package:lunch_me/... - as above, to be consistent across the codebase